src/patternfly/components/Page/themes/dark/page.scss

```diff
@@ -1,13 +1,36 @@
 @import "../../../../sass-utilities/themes/dark/all";
 
-@mixin pf-v5-theme-dark-page() {
-  .#{$page} {
-    --#{$page}__main-section--BackgroundColor: var(--#{$pf-global}--BackgroundColor--200);
-    --#{$page}__header-tools--c-button--m-selected--before--BackgroundColor: var(--#{$pf-global}--BackgroundColor--300);
-    --#{$page}__sidebar--BackgroundColor: var(--#{$pf-global}--BackgroundColor--100);
-    --#{$page}__header--BackgroundColor: var(--#{$pf-global}--palette--black-1000);
+@mixin pf-v5-theme-dark-vars {
+  --#{$page}__main-section--BackgroundColor: var(--#{$pf-global}--BackgroundColor--200);
+  --#{$page}__header-tools--c-button--m-selected--before--BackgroundColor: var(--#{$pf-global}--BackgroundColor--300);
+  --#{$page}__sidebar--BackgroundColor: var(--#{$pf-global}--BackgroundColor--100);
+  --#{$page}__header--BackgroundColor: var(--#{$pf-global}--palette--black-1000);
+
+  :where(.#{$page}__main-breadcrumb.pf-m-sticky-top),
+  :where(.#{$page}__main-breadcrumb.pf-m-sticky-bottom) {
+    --#{$page}__main-breadcrumb--BackgroundColor: var(--#{$pf-global}--BackgroundColor--300);
+  }
+
+  :where(.#{$page}__main-nav.pf-m-sticky-top),
+  :where(.#{$page}__main-nav.pf-m-sticky-bottom) {
+    --#{$page}__main-nav--BackgroundColor: var(--#{$pf-global}--BackgroundColor--300);
+  }
+
+  :where(.#{$page}__main-group.pf-m-sticky-top),
+  :where(.#{$page}__main-group.pf-m-sticky-bottom) {
+    --#{$page}__main-section--m-light--BackgroundColor: var(--#{$pf-global}--BackgroundColor--300);
+    --#{$page}__main-section--BackgroundColor: var(--#{$pf-global}--BackgroundColor--300);
+    --#{$page}__main-breadcrumb--BackgroundColor: var(--#{$pf-global}--BackgroundColor--300);
+    --#{$page}__main-nav--BackgroundColor: var(--#{$pf-global}--BackgroundColor--300);
   }
 
+  :where(.#{$page}__main-section.pf-m-sticky-top),
+  :where(.#{$page}__main-section.pf-m-sticky-bottom) {
```

Comment:
These `:where` selectors remove the specificity needed to override the original style rules. You can test this by adding `.pf-m-sticky-top` to a main section in this PR and on the main v5 branch. The background color of the page section should change.